components/bugzilla.py

```diff
@@ -32,7 +40,7 @@ def comment_on_bug(self, bug_id, status, try_run=None):
         if status == JOBSTATUS.COULD_NOT_VENDOR:
             comment = "./mach vendor failed with the following message: <TODO>"
         else:
-            comment = "I've submitted a try run for this commit: " + try_run
+            comment = "I've submitted a try run for this commit: https://treeherder.mozilla.org/#/jobs?repo=try&revision=" + try_run
```

Comment:
It comes from a database field which I think would be best of it was kept to bare commit ids.  

Which isn't a reason that I can't move the url out somewhere else and pass the full thing in; but doing that seems weird since I'll be putting some of the text of the comment in one place and some elsewhere....

Reply:
That makes sense! In that case I agree that how it is now is awesome